tests/test_app_config.py

```diff
@@ -24,7 +24,6 @@ def test_app_testing_config(self):
         self.assertFalse(application.config["DEBUG"])
         self.assertTrue(application.config["TESTING"])
         self.assertFalse(application.config["SQLALCHEMY_TRACK_MODIFICATIONS"])
-        self.assertEqual("sqlite://", application.config["SQLALCHEMY_DATABASE_URI"])
```

Comment:
Can we add similar line for `postgres`?

Reply:
I think it would depend on whether the developer is using `postgres` or `postgres+psycopg2`.

Reply:
@devkapilbansal , as per @epicadk said, this is very machine specific. To place it here in the test config means we have to provide separate syntax for each machines. Plus, we will ask contributors to always comment in/out the line/s that are not relevant to their machine when they run the local testing. After that, when they push the PR, they have to always only comment in the line that are going to be used when running test on CI (which is using ubuntu-latest) and comment out the rest, so, I don't think it's ideal.